libs/anglify/src/modules/dialog/dialog.service.ts

```diff
@@ -31,6 +31,7 @@ export class DialogService {
     scrollStrategy: this.overlay.scrollStrategies.block(),
     hasBackdrop: true,
     backdropClass: 'anglify-dialog-backdrop',
+    panelClass: 'anglify-dialog-pane',
```

Comment:
What is the panelClass responsible for?

Reply:
So the Dialog has a padding when there's a lot of content or 100% height and 100% width.